scripts/adapters/update-min-os-version.rb

```diff
@@ -4,12 +4,16 @@
 require 'json'
 require 'open3'
 
+def sdk_min_os_version
+  return Gem::Version.new('13.0')
```

Comment:
Can we fetch the version from the SDK podspec using `pod spec cat` similar to how we do for the partner SDK?
There should be something in common.rb to obtain the SDK pod name.
This version will change and we might forget to update it here. Also it may change at different points in time for Core and Mediation.,

Reply:
Yeah, I added this. It's a bit kludgy, but at this point, since we're actively migrating all of this over to the monorepo, I don't want to put a ton of effort into this.

Reply:
That's fair